packages/ui/src/theme/css/component/authenticator.scss

```diff
@@ -53,7 +53,7 @@
 
 [data-amplify-authenticator] [data-amplify-container] > [data-amplify-body] {
   background-color: var(--amplify-colors-background-primary);
-  box-shadow: var(--amplify-shadows-small);
+  box-shadow: var(--amplify-shadows-medium);
```

Comment:
The box shadow was barely visible on the docs site. This will make it more visible.